### PR TITLE
Fix update.sh

### PR DESCRIPTION
**Pull request by brendandburns**
@satnam6502 @filbranden

### CODE REVIEW

File: examples/update-demo/2-create-replication-controller.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ if [[ "${DOCKER_HUB_USER+set}" != "set" ]] ; then
   exit 1
 fi
 
-export KUBE_ROOT=$(dirname $0)/../..
+export KUBE_REPO_ROOT=$(dirname $0)/../..
```
Comment (filbranden) [inline examples/update-demo/2-create-replication-controller.sh]: The previous in #3576 actually said:

```shell
export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
```

Do we want that instead?

**Comment by filbranden** [comment]
One minor nit, otherwise looks good.

File: examples/update-demo/2-create-replication-controller.sh (hunk 1 above)
Comment (brendandburns) [inline examples/update-demo/2-create-replication-controller.sh]: done.

**Comment by brendandburns** [comment]
Comment addressed.  I'm merging since e2e is broken.

**Pull request merged**